tests/python/unittest/test_random.py

```diff
@@ -859,8 +859,8 @@ def test_randint_extremes():
     a = mx.nd.random.randint(dtype='int64', low=50000000, high=50000010, ctx=mx.context.current_context())
     assert a>=50000000 and a<=50000010
 
-@with_seed()
-@unittest.skip("Flaky test: https://github.com/apache/incubator-mxnet/issues/13446")
+# Seed set because the test is not robust enough to operate on random data
+@with_seed(1234)
```

Comment:
why fixed seed?

Reply:
yes now that PR #13543 is merged 
Confirmed that it resolved this flaky test
hence removing the fix seed